pittgoogle/registry_manifests/schemas.yml

```diff
@@ -33,11 +33,11 @@
   path: 'schemas/elasticc/elasticc.v0_9_1.brokerClassification.avsc'
 #
 # LSST alerts
-# [FIXME] Hacking in v7.1 for now. Need to figure out install lsst-alert-packet to use lsst_auto_schema_helper
+# [FIXME] Hacking in v7.3 for now. Need to figure out install lsst-alert-packet to use lsst_auto_schema_helper
 # - name: 'lsst.v<MAJOR>_<MINOR>.alert'
   # helper: 'lsst_auto_schema_helper'
-- name: 'lsst.v7_1.alert'
-  path: 'schemas/lsst/7/1/lsst.v7_1.alert.avsc'
+- name: 'lsst.v7_3.alert'
+  path: 'schemas/lsst/7/3/lsst.v7_3.alert.avsc'
```

Comment:
I think all three schema versions need to be registered here in order to be usable.